app/src/main/java/ch/epfl/skysync/dataModels/flightModels/Balloon.kt

```diff
@@ -0,0 +1,6 @@
+package ch.epfl.skysync.dataModels.flightModels
+
+data class Balloon(val name:String, val qualification: BalloonQualification)
```

Comment:
This should have an `id` field (even if never used by the front-end, it is needed for the database, especially in a case like `flight.setBalloon(balloon)`)